### PR TITLE
add `nodejs.eventloop.time` metric

### PR DESCRIPTION
**Pull request by maryliag**
## Changes

Adding `nodejs.eventloop.time` metric to a counter.
Feedback came during prototype of the nodejs runtime metrics [here](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2136/files#r1678358022).

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`

### CODE REVIEW

**Comment by maryliag** [comment]
cc @open-telemetry/javascript-approvers

File: docs/runtime/nodejs-metrics.md [hunk 1]
```diff
@@ -313,10 +313,10 @@ This metric is [recommended][MetricRecommended].
 
 | Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
 | -------- | --------------- | ----------- | -------------- | --------- |
-| `nodejs.eventloop.utilization` | Gauge | `1` | Event loop utilization. [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `nodejs.eventloop.utilization` | Counter | `s` | Event loop utilization. [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```
Comment (ben-childs-docusign) [inline docs/runtime/nodejs-metrics.md:316]: nodejs perf_hooks expose 2 cumulative counters to measure utilization:
https://nodejs.org/api/perf_hooks.html#performanceeventlooputilizationutilization1-utilization2

active seconds and idle seconds

the utilization value itself seems to somehow be computed based on the difference between two values and it is not clear if that could be represented as a counter directly.

my suggestion would be to add 
```
nodejs.eventloop.active.time
nodejs.eventloop.idle.time
```
and then keep utilization as a gauge if we think it is useful.

Comment (maryliag) [reply docs/runtime/nodejs-metrics.md]: Created a new metric for time, with the state as an attribute. Kept the utilization as is.

File: model/metrics/nodejs-metrics.yaml [hunk 2]
```diff
@@ -73,9 +73,23 @@ groups:
     type: metric
     metric_name: nodejs.eventloop.utilization
     brief: "Event loop utilization."
+    instrument: gauge
+    unit: "1"
+    stability: experimental
+    note: >
+      The value range is [0.0,1.0] and can be retrieved from `utilization` from
```
Comment (lmolkova) [inline model/metrics/nodejs-metrics.yaml:80]: nit 
```suggestion
      The value range is [0.0, 1.0] and can be retrieved from
```

File: model/metrics/nodejs-metrics.yaml [hunk 3]
```diff
@@ -73,9 +73,23 @@ groups:
...
     instrument: counter
     unit: "s"
     stability: experimental
+    attributes:
+      - ref: nodejs.eventloop.type
+        requirement_level: required
     note: >
-      Value can be retrieved from
-      [`performance.eventLoopUtilization([utilization1[, utilization2]])`](https://nodejs.org/api/perf_hooks.html#performanceeventlooputilizationutilization1-utilization2)
+      Value can be retrieved from `idle` and `active` from
```
Comment (lmolkova) [inline model/metrics/nodejs-metrics.yaml:94]: nit:

> Value can be retrieved from `idle` and `active` from
[...]()

this sentence seems to be broken

File: model/metrics/nodejs-metrics.yaml [hunk 4]
```diff
@@ -73,9 +73,23 @@ groups:
...
+      [`performance.eventLoopUtilization([utilization1[, utilization2]])`](https://nodejs.org/api/perf_hooks.html#performanceeventlooputilizationutilization1-utilization2)
```
Comment (lmolkova) [inline model/metrics/nodejs-metrics.yaml]:
```suggestion
      [`performance.eventLoopUtilization([utilization1[, utilization2]])`](https://nodejs.org/api/perf_hooks.html#performanceeventlooputilizationutilization1-utilization2)
```

File: model/metrics/nodejs-metrics.yaml [hunk 5]
```diff
@@ -73,9 +73,23 @@ groups:
...
+
+  - id: metric.nodejs.eventloop.time
+    type: metric
+    metric_name: nodejs.eventloop.time
+    brief: "Cumulative duration of time the event loop has been on the state."
     instrument: counter
     unit: "s"
     stability: experimental
+    attributes:
+      - ref: nodejs.eventloop.type
```
Comment (lmolkova) [inline model/metrics/nodejs-metrics.yaml:91]: I don't feeling strong about it, but please consider `nodejs.eventloop.state` which seems to work better with `active` and `idle` values. I'd expect something else under type (like specific implementation name).

File: model/registry/nodejs.yaml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+        type:
+          members:
+            - id: active
+              value: 'active'
+              brief: 'Active time.'
+              stability: experimental
+            - id: idle
+              value: 'idle'
+              brief: 'Idle time.'
+              stability: experimental
```
Comment (lmolkova) [inline model/registry/nodejs.yaml]:
```suggestion
              stability: experimental

```

File: model/metrics/nodejs-metrics.yaml (hunk 5 above)
Comment (david-luna) [reply model/metrics/nodejs-metrics.yaml:91]: +1 to the naming change. Looking at some refs within semconv I see the concept of `mode` for [cpu time](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/system-metrics.md#metric-systemcputime) or `state` for [memory usage](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/system-metrics.md#metric-systemcputime)

Comment (maryliag) [reply model/metrics/nodejs-metrics.yaml]: Updated

File: model/metrics/nodejs-metrics.yaml [hunk 7]
```diff
@@ -77,5 +77,19 @@ groups:
...
     stability: experimental
     note: >
-      The value range is [0.0,1.0] and can be retrieved from value
+      The value range is [0.0, 1.0] and can be retrieved from
+      [`performance.eventLoopUtilization([utilization1[, utilization2]])`](https://nodejs.org/api/perf_hooks.html#performanceeventlooputilizationutilization1-utilization2)
+
+  - id: metric.nodejs.eventloop.time
+    type: metric
+    metric_name: nodejs.eventloop.time
+    brief: "Cumulative duration of time the event loop has been on the state."
```
Comment (lmolkova) [inline model/metrics/nodejs-metrics.yaml:86]: maybe

```suggestion
    brief: "Cumulative duration of time the event loop has been in each state."
```

?

**Review by lmolkova — APPROVED** [review]

**Review by jsuereth — APPROVED** [review]

File: model/registry/nodejs.yaml [hunk 8]
```diff
@@ -0,0 +1,20 @@
+groups:
+  - id: registry.nodejs
+    type: attribute_group
+    brief: Describes Node.js related attributes.
+    display_name: Node.js Attributes
+    prefix: nodejs
```
Comment (lmolkova) [inline model/registry/nodejs.yaml:6]: we've got rid of prefix while you were working on this PR, so this should be removed and added to the attribute id

```suggestion
```

File: model/registry/nodejs.yaml [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    attributes:
+      - id: eventloop.state
```
Comment (lmolkova) [inline model/registry/nodejs.yaml:8]:
```suggestion
      - id: nodejs.eventloop.state
```

**Pull request merged**